<launch>

  <node pkg="franka_action_lib" type="execute_skill_action_server_node" name="execute_skill_action_server_node" output="screen">
    <param name="publish_frequency" value="10" />
  </node>

  <node pkg="franka_action_lib" type="robot_state_publisher_node" name="robot_state_publisher_node" output="screen">
    <param name="publish_frequency" value="100" />
  </node>
  
  <node pkg="franka_action_lib" type="robolib_status_publisher_node" name="robolib_status_publisher_node" output="screen">
    <param name="publish_frequency" value="100" />
  </node>
  
  <node pkg="franka_action_lib" type="get_current_robot_state_server_node" name="get_current_robot_state_server_node" output="screen">
    <param name="robot_state_topic_name_" value="/robot_state_publisher_node/robot_state" />
  </node>

  <node pkg="franka_action_lib" type="get_current_robolib_status_server_node" name="get_current_robolib_status_server_node" output="screen">
    <param name="robot_state_topic_name_" value="/robolib_status_publisher_node/robolib_status" />
  </node>

</launch>